### DISCUSSION TITLE
ML4W Excellence Available on Fedora (Nobara)?

### DISCUSSION THREAD

**Discussion by jamesmac10** [mylinuxforwork/dotfiles#408] [General]
Hi there,

I've really enjoyed using the My Linux 4 Work starter and dotfiles packages on Arch, now I would really like to replicate the features etc on Fedora. Nobara specifically. I know the starter package works well on Fedora, but is there a way to get the dotfiles installer running on Fedora also? I love Arch but Fedora is my usual daily. Thanks

**Comment by dwilliam62** [comment]
The full environment is not yet available for Fedora/Nobara.   He has a starter kit for Fedora

**Comment by mylinuxforwork** [comment]
The support for Fedora is coming very soon. Working on the last improvements of the installer before releasing the first beta.

**Comment by jamesmac10** [comment]
Oh awesome. Really looking forward to it! super exciting news thank you.

**Comment by Bammede** [comment]
Please add ML4W dotfiles to Fedora COPR repos (it is like AUR for Arch).

**Reply by mylinuxforwork** [reply]
It’ll be not needed with the new Dotfiles Installer. Will post a video soon.
